### DISCUSSION TITLE
Help, Canned Message Module Disabled

### DISCUSSION THREAD

**Discussion by Dogsrearwheels** [meshtastic/firmware#5567] [Q&A]
Hi I am a newbie to the project, I have connected a rotary encoder, checked the wiring, checked the settings, umpteen times, still the message ‘canned message module disabled’ even though enabled in app? Can anybody help?![image](https://github.com/user-attachments/assets/e4fa7cef-85a4-4c77-a158-48a525740f8a)![image](https://github.com/user-attachments/assets/59f0ed06-7a4e-4182-9e3c-9aff66616864)

**Comment by Visnitz** [comment]
Have a same problem with heltec V3. Try  on 2.5 and 2.6 firmware too. :(

**Comment by MomentumV** [comment]
There's apparently a bug in the IOS app. 
It's showing the disabled message because the app won't let you save the canned messages string.
@Visnitz  the answer (use android or CLI to define the canned messages) was given in this reddit thread on the same issue. https://www.reddit.com/r/meshtastic/comments/1heptid/help_canned_message_module_disabled/

**Reply by dgtlrift** [reply]
So where do we file bugs for the iOS Meshtastic app?

**Reply by MomentumV** [reply]
not sure! sorry! Just sharing what I had found.
